odoo/addons/base/wizard/base_module_uninstall.py

```diff
@@ -52,12 +52,12 @@ def lost(model):
                 # find the models that have all their XIDs in the given modules
                 self.model_ids = ir_models.filtered(lost).sorted('name')
 
-    @api.onchange('module_id')
+    @api.onchange('module_ids')
     def _onchange_module_id(self):
         # if we select a technical module, show technical modules by default
-        if not self.module_id.application:
+        if not self.module_ids.application:
```

Comment:
Please adapt the comment message: `if we select technical modules only, show technical modules`.